vdev/src/git.rs

```diff
@@ -87,6 +87,44 @@ pub fn get_modified_files() -> Result<Vec<String>> {
     Ok(capture_output(&args)?.lines().map(str::to_owned).collect())
 }
 
+pub fn set_config_values(config_values: &[(&str, &str)]) -> Result<String> {
+    let mut args = vec!["config"];
+
+    for (key, value) in config_values {
+        args.push(key);
+        args.push(value);
+    }
+
+    capture_output(&args)
+}
+
+/// Checks if the current directory's repo is clean
+pub fn check_git_repository_clean() -> Result<bool> {
+    Ok(Command::new("git")
+        .args(["diff-index", "--quiet", "HEAD"])
+        .stdout(std::process::Stdio::null())
+        .status()
+        .map(|status| status.success())?)
+}
+
+/// Commits changes from the current repo
+pub fn commit(commit_message: &str) -> Result<String> {
+    Command::new("git")
+        .args(["-am", commit_message])
+        .capture_output()
```

Comment:
With the recent change, I think this can just be:
```suggestion
    capture_output(&["am", commit_message])
```

Reply:
As discussed, capture_output will work regardless of directory